.zenodo.json

```diff
@@ -93,6 +93,11 @@
       "orcid": "0000-0001-9813-3167",
       "type": "ProjectMember"
     },
+    {
+      "name": "Trievel, Alec B.",
+      "affiliation": "Open source developer/contributor",
+      "type": "ProjectMember"
+    },
```

Comment:
I placed you above the "last author". I wasn't explicit about it, but I'm treating this like an academic paper where first and last authors are special positions.